conf/modules.config

```diff
@@ -489,6 +489,18 @@ process {
 
     }
 
+    withName: HRDCNA {
+        publishDir = [
+            path: { "${params.outdir}/hrdcna/" },
+            mode: params.publish_dir_mode,
+            saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
+        ]
```

Comment:
We might use `ext.when` here:

`ext.when = { params.hrdcna_compute_score && params.caller == "ascat_sc" },`